opentidalfarm/problems/steady_sw.py

```diff
@@ -75,6 +76,11 @@ class SteadySWProblemParameters(FrozenClass):
     # Boundary conditions
     bcs = BoundaryConditionSet()
 
+    # Used in order to avoid machine precision error when calcualting number of 
+    # timesteps for dynamic friction. 
+    def finished(self, current_time, finish_time):
+        return float(current_time - finish_time) >= - 1e3*DOLFIN_EPS
+
```

Comment:
Move this to sw.py and multi_steady_sw.py and replace it here with a simple "return True" function?